code/game/gamemodes/gameticker.dm

```diff
@@ -376,6 +383,20 @@ var/round_start_time = 0
 			if(!istype(M,/mob/new_player))
 				to_chat(M, "Captainship not forced on anyone.")
 
+/datum/controller/gameticker/proc/send_tip_of_the_round()
+	var/m

```

Comment:
This function is almost the exact same as `/client/verb/randomtip`. Might be worth seeing if you can just make the target of the tip an argument to avoid code duplication.

Reply:
I will keep them seperated for now, as I can see someone will want to tweak only the random tip OOC verb in the future (for example removing silly tips change from it, or changing output message color), and output message differs slightly.